docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+ SERVICES      <list of your service's names>
+ APP URLS      http://localhost:62177
+               http://<app name>.lndo.site/
+               https://<app name>.lndo.site/
+               http://www.<app name>.lndo.site/
+               https://www.<app name>.lndo.site/
 ```
 
-Where the value after `-r` is the relationship name in the application for the service you want to download, and `-m` is the Platform.sh mount path you want to download.  You may specify as many relationships (multiple `-r` switches) and as many mounts (multiple `-m` switches) as you wish.
+Access your app and services by opening the `APP URLS` in your browser.
+
+For more examples, see the [getting started page](https://docs.lando.dev/platformsh/getting-started.html).
+
+## Configuring lando
+
+Lando configuration is located in [the `.lando.yml` file](https://docs.lando.dev/platformsh/config.html).
+
+If you changed your [`.platform.app.yaml`](../../configuration/app/_index.md), [`.platform/routes.yaml`](../../configuration/routes/_index.md) configuration files, run `lando rebuild` for the changes to be taken into account.
+
+### Untrusted SSL certificate
+
+By default, when accessing your local Lando sites through `https` you get an error message in your browser.
+This is expected behavior:
+
+> Lando uses its own Certificate Authority (CA) to sign the certificates for each service and to ensure that these certs are trusted on the internal Lando network.
+However, while Lando will automatically trust this CA internally it's up to you to trust it on your host machine. Doing so alleviates browser warnings regarding the certificates issued.
+
+Find out how to solve it in [Lando's Blog](https://lando.dev/blog/2020/03/20/_5-things-to-do-after-you-install-lando/).
+
+## Common tasks
+
+**TODO: clarify that part + test**
+
+- [Update](https://docs.lando.dev/help/updating.html#_1-do-a-lando-rebuild)
+- [Sync](https://docs.lando.dev/platformsh/sync.html)
+
+More usage examples of the `.lando.yml` file [can be found in GitHub](https://github.com/lando/platformsh/tree/main/examples)
+
+## Troubleshooting
+
+- Make sure that the platform config files (`.platform.app.yaml`, `.platform/routes.yaml`) are present in your local repository.
+- Check that the [services and languages](https://docs.lando.dev/platformsh/config.html#services-yaml) defined in your `.platform.app.yaml` are supported by Lando.
+- Check the [caveats and known issues](https://docs.lando.dev/platformsh/caveats.html).
+- Carefully check the output of the Lando commands you run to spot warnings and errors.
+- Run `lando rebuild`.
+- Restart Lando in debug mode `lando restart -vvv`.
+- Check that you don't face a common issues, as the [dns-rebind](https://docs.lando.dev/help/dns-rebind.html) for example.
+- Check the [Lando documentation](https://docs.lando.dev/help/logs.html#install-logs) for more extensive troubleshooting.
+
+### Access the logs
+
+Run `lando list` to get the list of services.
```

Comment:
```suggestion
Run `lando list` to get a list of services.
```
I would not start with this as without context it's not clear why it's present. Maybe it would help if it was `a list of services you're using` or something.